client/js/views/MyProfile.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <v-container class="profile">
+    <h1>{{ user }} Profile Settings</h1>
+    <br />
+    <v-text-field
+      v-model="email"
+      outlined
+      :dense="$vuetify.breakpoint.mdAndDown ? true : false"
+      hint="email"
+      data-cy="email"
+      autocomplete="email"
+      placeholder="email"
+      @keydown.enter.prevent="submitEmail"
+    />
+    <v-snackbar
+      v-model="showSnackBar"
+      :color="colorValue"
+      content-class="d-flex justify-space-between align-center"
+      data-cy="auth-snackbar"
+    >
+      {{ snackBarMessage }}
+      <v-icon data-cy="close-snackbar" @click="clearSnackBar"> mdi-close </v-icon>
+    </v-snackbar>
+    <!-- <h3>Current Email: test</h3> -->
+  </v-container>
+</template>
+<script>
+export default {
+  name: 'MyProfile',
+  data() {
+    return {
+      user: this.$store.state.auth.username,
+      email: '',
+      showSnackBar: false,
+      snackBarMessage: '',
+      colorValue: '',
+    };
+  },
+  computed: {},
+  mounted() {},
+  methods: {
+    async submitEmail() {
+      try {
+        //Email Testing Regex
+        if (/^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/.test(this.email)) {
```

Comment:
Let's rewrite the validity check into a computed property on the stored input value and then disable submission with an error message on the input when the email is invalid